v2/sourcedb-to-spanner/README_Sourcedb_to_Spanner.md

```diff
@@ -121,49 +121,72 @@ need valid resources for the required parameters.
 Provided that, the following command line can be used:
 
 ```shell
-export PROJECT=<my-project>
-export BUCKET_NAME=<bucket-name>
-export REGION=us-central1
-export TEMPLATE_SPEC_GCSPATH="gs://$BUCKET_NAME/templates/flex/Sourcedb_to_Spanner_Flex"
-
-### Required
-export INSTANCE_ID=<instanceId>
-export DATABASE_ID=<databaseId>
-export PROJECT_ID=<projectId>
-
-### Optional
-export DRIVER_JARS=<driverJars>
-export DRIVER_CLASS_NAME=<driverClassName>
-export CONNECTION_URL=<connectionURL>
-export CONNECTION_PROPERTIES=<connectionProperties>
-export USERNAME=<username>
-export PASSWORD=<password>
-export TABLES=<tables>
-export NUM_PARTITIONS=<numPartitions>
-export SPANNER_HOST=https://batch-spanner.googleapis.com
-export IGNORE_COLUMNS=<ignoreColumns>
-export DISABLED_ALGORITHMS=<disabledAlgorithms>
-export EXTRA_FILES_TO_STAGE=<extraFilesToStage>
-
-gcloud dataflow flex-template run "sourcedb-to-spanner-flex-job" \
-  --project "$PROJECT" \
-  --region "$REGION" \
-  --template-file-gcs-location "$TEMPLATE_SPEC_GCSPATH" \
-  --parameters "jdbcDriverJars=$DRIVER_JARS" \
-  --parameters "jdbcDriverClassName=$DRIVER_CLASS_NAME" \
-  --parameters "sourceConnectionURL=$CONNECTION_URL" \
-  --parameters "sourceConnectionProperties=$CONNECTION_PROPERTIES" \
-  --parameters "username=$USERNAME" \
-  --parameters "password=$PASSWORD" \
-  --parameters "tables=$TABLES" \
-  --parameters "numPartitions=$NUM_PARTITIONS" \
-  --parameters "instanceId=$INSTANCE_ID" \
-  --parameters "databaseId=$DATABASE_ID" \
-  --parameters "projectId=$PROJECT_ID" \
-  --parameters "spannerHost=$SPANNER_HOST" \
-  --parameters "ignoreColumns=$IGNORE_COLUMNS" \
-  --parameters "disabledAlgorithms=$DISABLED_ALGORITHMS" \
-  --parameters "extraFilesToStage=$EXTRA_FILES_TO_STAGE"
+# Adding timestamp just allows the name to be unique across multiple runs
+JOB_NAME="<job-name>-`date +'%Y-%m-%d-%H-%M-%S-%N'`"
```

Comment:
Did you edit this manually? I think some of this documentation is autogenerated. Please check that your changes do not get overwritten during auto generation!

Reply:
+1.

I don't think `curl` is part of provided that. The auto generated provided that is a env setting step for the `flex template run`, `curl` is an alternative way of doing the same thing.

Reply:
This file is generated manually using :
https://github.com/GoogleCloudPlatform/DataflowTemplates/blob/main/contributor-docs/code-contributions.md#generated-documentation

Reply:
Updated from the auto gen docs.